locales/en-US/server.ftl

```diff
@@ -280,6 +280,7 @@ settingsDescription = You can sign in with Firefox Accounts to sync all your scr
 settingsPageSubHeader = Sync & Accounts
```

Comment:
This should not be Sync and Accounts...
It should just be `Firefox Account`

Reply:
We should not use the word Sync here, Sync is a specific service name and we should not conflate the two